.github/workflows/lint.yml

```diff
@@ -10,8 +10,8 @@ on:
 
 env: # Comment env block if you do not want to apply fixes
   # Apply linter fixes configuration
-  APPLY_FIXES: all # When active, APPLY_FIXES must also be defined as environment variable (in github/workflows/mega-linter.yml or other CI tool)
-  APPLY_FIXES_EVENT: all # Decide which event triggers application of fixes in a commit or a PR (pull_request, push, all)
+  # APPLY_FIXES: all # When active, APPLY_FIXES must also be defined as environment variable (in github/workflows/mega-linter.yml or other CI tool)
```

Comment:
I disabled megalinter attempts to create PRs or commits as it will never work.
If you want you can download megalinter artifacts and apply code formatting / address findings in followup commit.